From f4bf264837a28342bc86885138e2a50ebd5ea4cf Mon Sep 17 00:00:00 2001
From: Neil Cheng <Neil.Cheng@quantatw.com>
Date: Fri, 14 Aug 2026 18:30:59 +0800
Subject: [PATCH] configurations: rainier: Remove CPU MCTPI3CTarget

The CPU MCTPI3CTarget configuration was added before the CPU provided
its Terminus Name through PLDM.

Starting from SoC FW 0.5.2, the CPU PLDM PDRs provide the Terminus Name,
allowing the CPU terminus to be identified without this static
entity-manager configuration.

Remove the obsolete CPU MCTPI3CTarget entry.

Upstream-Status: Submitted [https://gerrit.openbmc.org/c/openbmc/entity-manager/+/93541]

Change-Id: I62c18b359806bbc78ad3e600ff66afe46761d600
Signed-off-by: Neil Cheng <Neil.Cheng@quantatw.com>
---
 .../meta/rainier/rainier_hpm_vr_mps_v1p2vr_tic.json | 13 -------------
 .../meta/rainier/rainier_hpm_vr_rtt_v1p2vr_mps.json | 13 -------------
 2 files changed, 26 deletions(-)

diff --git a/configurations/meta/rainier/rainier_hpm_vr_mps_v1p2vr_tic.json b/configurations/meta/rainier/rainier_hpm_vr_mps_v1p2vr_tic.json
index 01a1b46..8801541 100644
--- a/configurations/meta/rainier/rainier_hpm_vr_mps_v1p2vr_tic.json
+++ b/configurations/meta/rainier/rainier_hpm_vr_mps_v1p2vr_tic.json
@@ -476,19 +476,6 @@
             "Name": "Rainier_MB_CPLD",
             "Type": "LatticeLCMXO3D_9400Firmware"
         },
-        {
-            "Address": [
-                85,
-                231,
-                18,
-                52,
-                86,
-                120
-            ],
-            "Bus": "3",
-            "Name": "CPU0",
-            "Type": "MCTPI3CTarget"
-        },
         {
             "Address": "0x60",
             "Bus": 11,
diff --git a/configurations/meta/rainier/rainier_hpm_vr_rtt_v1p2vr_mps.json b/configurations/meta/rainier/rainier_hpm_vr_rtt_v1p2vr_mps.json
index 33a0a08..121239a 100644
--- a/configurations/meta/rainier/rainier_hpm_vr_rtt_v1p2vr_mps.json
+++ b/configurations/meta/rainier/rainier_hpm_vr_rtt_v1p2vr_mps.json
@@ -476,19 +476,6 @@
             "Name": "Rainier_MB_CPLD",
             "Type": "LatticeLCMXO3D_9400Firmware"
         },
-        {
-            "Address": [
-                85,
-                231,
-                18,
-                52,
-                86,
-                120
-            ],
-            "Bus": "3",
-            "Name": "CPU0",
-            "Type": "MCTPI3CTarget"
-        },
         {
             "Address": "0x76",
             "Bus": 11,
